### DISCUSSION TITLE
LibreY with tor and HTTP(s)

### DISCUSSION THREAD

**Discussion by robert-winkler** [Ahwxorg/LibreY#200] [Show and tell]
Hi, I just managed to configure the following functions of my  <https://libre-find.online> instance:

- On Clearnet, HTTP is forwarded to HTTPS.
- The Cleanet browser informs about the presence of a Tor address.
- Tor connects to HTTP but is also HTTPS capable.

All possible connections are encrypted and use a Let's Encrypt certificate.
I used the excellent guide of <https://blog.pastly.net/posts/2021-05-03-website-setup/>
Have fun!

**Comment by ghost** [comment]
Only specific certificate authorities (usually paid) issue certificates for `.onion` addresses.

They're also kind of pointless anyway:
hxxps://community[.]torproject[.]org/onion-services/advanced/https/
hxxp://xmrhfasfg5suueegrnc4gsgyi2tyclcy5oz7f5drnrodmdtob6t2ioyd[.]onion/onion-services/advanced/https/
